services/movie.py

```diff
@@ -0,0 +1,36 @@
+from typing import List, Optional
+
+from django.db.models.query import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None) -> QuerySet:
+    result_query = Movie.objects.all()
+
+    if genres_ids:
+        result_query = result_query.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        result_query = result_query.filter(actors__id__in=actors_ids)
+
+    return result_query
+
+
+def get_movie_by_id(movie_id: int) -> Optional[Movie]:
+    if not movie_id:
```

Comment:
There is no need for this - here would be fine just to return `Movie.objects.get(id=movie_id)` and if error happens you would catch it wherever you call this service